src/igvfd/tests/fixtures/schemas/sequence_file.py

```diff
@@ -264,3 +264,15 @@ def sequence_file_v12(sequence_file):
         'schema_version': '12'
     })
     return item
+
+
+@pytest.fixture
+def sequence_file_v14(sequence_file_v12):
+    item = sequence_file_v12.copy()
```

Comment:
I'd suggest just copying the base fixture sequence_file for the same reason as above